src/nn/Modules/Activations.cpp

```diff
@@ -74,18 +71,18 @@ namespace af
 
         Variable ELU::forward(const Variable &input)
         {
-            auto res = max(input, m_alpha * (exp(input) - 1));
-            return res;
+            auto mask = input >= 0.0;
+            return (mask * input) + (!mask * m_alpha * exp(input));
```

Comment:
Shouldn't this be `(!mask * m_alpha * (exp(input) - 1))` ?
https://keras.io/layers/advanced-activations/#elu